jujugui/static/gui/src/app/app.js

```diff
@@ -1128,11 +1133,22 @@ YUI.add('juju-gui', function(Y) {
         });
         return;
       }
+      const beta = window.flags && window.flags.beta;
+      const flow = metadata && metadata.activeComponent !== 'flow';
+      if (beta && !flow && !modelCommitted) {
```

Comment:
I'm going to side with @frankban, since we seem to be interested in it not being "flow", and "flow" is actually the case of the component not being flow?